src/Components/Form/Form.tsx

```diff
@@ -0,0 +1,47 @@
+import { Container, Button } from '@mui/material'
+import { useState } from 'react'
+import { NumberInputTextField } from '../NumberInputTextField/NumberInputTextField'
+
+export function Form(): JSX.Element {
+    const [focusTime, setFocusTime] = useState<number>(25)
+    const [brakeTime, setBrakeTime] = useState<number>(5)
+    const [longerBrakeTime, setLongerBrakeTime] = useState<number>(15)
```

Comment:
You can skip `number` typings. TypeScript will implicitly set those variables' types to numbers if their default values are so.